### ISSUE TITLE
Has `AccessibilityInfo` Error when build ios

### ISSUE THREAD

**Issue by Kennytian** [facebook/react-native#14220]
### good work on react-native-router-flux v3.39.1  +  react-native v0.44.1  +  react v16.0.0-alpha.6

Version

Tell us which versions you are using:

react-native-router-flux v3.39.1
react-native v0.44.2
react v16.0.0-alpha.6
Expected behaviour

Build successfully.

Actual behaviour
Go error

error: bundling: UnableToResolveError: Unable to resolve module `AccessibilityInfo` from `/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map or in these directories:
  /Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/node_modules
,   /Users/kenny/projects/work/nutfin/almondWallet/node_modules

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset packager cache: `rm -fr $TMPDIR/react-*` or `npm start -- --reset-cache`.
    at UnableToResolveError (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:525:5)
    at ResolutionRequest._resolveNodeDependency (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:394:11)
    at tryResolveSync (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:136:20)
    at tryResolveSync (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:66:12)
    at ResolutionRequest.resolveDependency (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:134:22)
    at dependencyNames.map.name (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:145:59)
    at Array.map (native)
    at ResolutionRequest.resolveModuleDependencies (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:145:42)
    at module.read.then (/Users/kenny/projects/work/nutfin/almondWallet/node_modules/react-native/packager/src/node-haste/DependencyGraph/ResolutionRequest.js:172:40)
    at process._tickCallback (internal/process/next_tick.js:109:7)
Bundling `index.ios.js`  98.7% (599/603), failed.

## same issues  from https://github.com/aksonov/react-native-router-flux/issues/1903

**Comment by hramos** [comment]
Hey, thanks for reporting this issue!

It looks like your description is missing some necessary information. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md)? This is necessary for people to be able to understand and reproduce the issue being reported.

I am going to close this, but feel free to open a new issue with the additional information provided. Thanks!

**Comment by sillybill** [comment]
any solutions?

**Comment by rajatyadav** [comment]
Looking for solution for the same problem.

**Comment by karna41317** [comment]
```

"react-native": "^0.47.2",
"react-dom": "16.0.0-alpha.12",
"react": "16.0.0-alpha.12",

```
Cleared cache, 
`rm -rf node-modules, npm install`
`yarn cache clean, yarn start`

Still same, local builds OKay, but when i build on build server, i get this issue, 

Also i am not using  react-native-router-flux. 

this is something to do with react-native, 

this bug stopping everyone in my team. ready to provide more info
